base/src/main/java/com/gdgistanbul/projectx/android/BaseActivity.kt

```diff
@@ -15,4 +19,8 @@ abstract class BaseActivity : AppCompatActivity() {
         AndroidInjection.inject(this)
         super.onCreate(savedInstanceState)
     }
+
+    inline fun FragmentManager.inTransaction(func: FragmentTransaction.() -> FragmentTransaction) {
+        beginTransaction().func().commit()
```

Comment:
should move into an extension file.  I think let' s try to create a method to replace the fragment.  https://proandroiddev.com/kotlin-extensions-to-commit-fragments-safely-de06218a1f4